packages/fast-jss-manager-react/src/manage-jss.tsx

```diff
@@ -106,11 +105,11 @@ function manageJss<S, C>(styles?: ComponentStyles<S, C>): <T>(Component: React.C
                 // On construction, check if the style object or component object have already been used.
                 // If not, we need to store them in our weakmaps for later use
                 if (!Boolean(JSSManager.styleMap.get(styles))) {
-                    JSSManager.styleMap.set(styles, uuid());
+                    JSSManager.styleMap.set(styles, uniqueId());
```

Comment:
I'm thinking we should probably prefix where we use this so we don't just get `id="1"` - maybe prefix with the package name? or just `fast-`. Thoughts?

Reply:
nm this is probably fine, this doesn't have anything to do with DOM

Reply:
I think as a principle though we for sure should. 

We can prefix with fast, but I would add context as well (a la the dev site instance above).

```
uniqueId(`fast${context}`)
```